package.json

```diff
@@ -1,4 +1,94 @@
 {
   "name": "builder-react-component",
-  "version": "0.0.0"
+  "version": "0.0.1",
+  "description": "Builder Archetype - React Component",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FormidableLabs/builder-react-component.git"
+  },
+  "author": "Ryan Roemer <ryan.roemer@formidablelabs.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/FormidableLabs/builder-react-component/issues"
+  },
+  "homepage": "https://github.com/FormidableLabs/builder-react-component",
+  "scripts": {
+    "npm:postinstall": "node -e \"require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})\" || builder run build-lib",
+    "npm:preversion": "builder run check",
+    "npm:version": "builder run clean && builder run build && git add -A dist",
+    "npm:test": "builder run test-frontend",
+    "clean-dist": "rimraf dist",
+    "build-dist-min": "webpack --config node_modules/builder-react-component/config/webpack/webpack.config.js",
```

Comment:
I strongly prefer leaving it really explicit, and based on overriding specific npm scripts rather than just masking files. That way you only need to look in `package.json` to determine whether you're getting the "off the shelf" behavior or not. And if you do need to change something, the process is no different than writing a regular npm script, with no strange gotchas based on how you name a file or where you put it.